README.md

```diff
@@ -24,21 +24,21 @@ We felt it important to announce this, particularly in view of [the dockershim r
 
-If you have any concerns or questions, we will be here to answer them in [issues, discussions, and/or on slack](#communication). Below you will find information / detail about our [CRI Integration](#cri) implementation.
+If you have any concerns or questions, we will be here to answer them in [issues, discussions, and/or on slack](#communication). Below you will find information/detail about our [CRI Integration](#cri) implementation.
 
 For containerd users already on v1.6.0-v1.6.3, there are known issues addressed by [v1.6.4](https://github.com/containerd/containerd/releases/tag/v1.6.4). The issues are primarily related to [CNI setup](https://github.com/kubernetes/website/blob/dev-1.24/content/en/docs/tasks/administer-cluster/migrating-from-dockershim/troubleshooting-cni-plugin-related-errors.md)
 
 ### Now Recruiting
 
 We are a large inclusive OSS project that is welcoming help of any kind shape or form:
 * Documentation help is needed to make the product easier to consume and extend.
-* We need OSS community outreach / organizing help to get the word out; manage
-and create messaging and educational content; and to help with social media, community forums/groups, and google groups.
+* We need OSS community outreach/organizing help to get the word out; manage
+and create messaging and educational content; and help with social media, community forums/groups, and google groups.
 * We are actively inviting new [security advisors](https://github.com/containerd/project/blob/main/GOVERNANCE.md#security-advisors) to join the team.
 * New sub-projects are being created, core and non-core that could use additional development help.
 * Each of the [containerd projects](https://github.com/containerd) has a list of issues currently being worked on or that need help resolving.
-  - If the issue has not already been assigned to someone, or has not made recent progress and you are interested, please inquire.
-  - If you are interested in starting with a smaller / beginner level issue, look for issues with an `exp/beginner` tag, for example [containerd/containerd beginner issues.](https://github.com/containerd/containerd/issues?q=is%3Aissue+is%3Aopen+label%3Aexp%2Fbeginner)
+  - If the issue has not already been assigned to someone or has not made recent progress and you are interested, please inquire.
+  - If you are interested in starting with a smaller / beginner-level issue, look for issues with an `exp/beginner` tag, for example [containerd/containerd beginner issues.](https://github.com/containerd/containerd/issues?q=is%3Aissue+is%3Aopen+label%3Aexp%2Fbeginner)
```

Comment:
We did the same for the others so lets do `smaller/beginner-level` here (no space between).

Reply:
Fixed!

Reply:
Thanks!